references/cli/status-ui/model/k8s_object.go

```diff
@@ -0,0 +1,87 @@
+package model
+
+import (
+	"context"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/oam-dev/kubevela/pkg/velaql/providers/query"
+)
+
+type K8SObject struct {
+	name       string
+	namespace  string
+	kind       string
+	apiVersion string
+	cluster    string
+	status     string
+}
+
+type K8SObjectList struct {
+	title []string
+	data  []K8SObject
+}
+
+func (l *K8SObjectList) Header() []string {
+	return l.title
+}
+
+func (l *K8SObjectList) Body() [][]string {
+	data := make([][]string, 0)
+	for _, app := range l.data {
+		data = append(data, []string{app.name, app.namespace, app.kind, app.apiVersion, app.cluster, app.status})
+	}
+	return data
+}
+
+func ListObjects(ctx context.Context, c client.Client) *K8SObjectList {
+	list := &K8SObjectList{
+		title: []string{"Name", "Namespace", "Kind", "APIVersion", "Cluster", "Status"},
+	}
+	name, ok := ctx.Value("appName").(string)
+	if !ok {
+		return list
+	}
+	namespace, ok := ctx.Value("appNs").(string)
+	if !ok {
+		return list
+	}
+	cluster, ok := ctx.Value("cluster").(string)
+	if !ok {
+		return list
+	}
+	opt := query.Option{
+		Name:      name,
+		Namespace: namespace,
+	}
+	if cluster != "local" {
```

Comment:
There is one const named "ClusterLocalName" in multicluster.go。 It is better to use it , not String "local"

Reply:
ok, I have fixed it.